src/CampaignAdmin.php

```diff
@@ -287,14 +288,24 @@ protected function getChangeSetItemResource(ChangeSetItem $changeSetItem)
             $referencedBy = [];
             foreach ($referencedItems as $referencedItem) {
                 $referencedBy[] = [
-                    'href' => $this->SetLink($referencedItem->ID)
+                    'href' => $this->SetLink($referencedItem->ID),
+                    'ObjectID' => $referencedItem->ObjectID
                 ];
             }
             if ($referencedBy) {
-                $hal['_links']['referenced_by'] = $referencedBy;
+                $hal['links']['referenced_by'] = $referencedBy;
             }
         }
 
+        $referToItems = $changeSetItem->findReferenced();
```

Comment:
@tractorcow That does not work. `findReferened()` get all implicit **objects** for this change while `References()` is a list of implicit **items** required by this change.